upgrade/1.2/scripts/upgrade/prerequisites.sh

```diff
@@ -573,6 +573,30 @@ if [ "$?" -eq 0 ]; then
 fi
 set -e
 
+state_name="ADD_MTL_ROUTES"
+state_recorded=$(is_state_recorded "${state_name}" $(hostname))
+if [[ $state_recorded == "0" && $(hostname) == "ncn-m001" ]]; then
+    echo "====> ${state_name} ..."
+
+    export PDSH_SSH_ARGS_APPEND="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+    NCNS=$(grep -oP 'ncn-w\w\d+|ncn-s\w\d+' /etc/hosts | sort -u)
+    Ncount=$(echo $NCNS | wc -w)
+    HOSTS=$(echo $NCNS | tr -t ' ' ',')
+    pdsh -w $HOSTS ip route add 10.1.0.0/16 via 10.252.0.1 dev vlan002
```

Comment:
These probably shouldn't be hard-coded.

Reply:
Yup, @johren-hpe mentioned that in the JIRA.